test/recipes/95-test_external_tlsfuzzer.t

```diff
@@ -18,8 +18,8 @@ plan skip_all => "No external tests in this configuration"
     if disabled("external-tests");
 plan skip_all => "TLSFuzzer tests not available on Windows or VMS"
     if $^O =~ /^(VMS|MSWin32)$/;
-plan skip_all => "TLSFuzzer tests not supported in out of tree builds"
-    if bldtop_dir() ne srctop_dir();
+plan skip_all => "TLSFuzzer is not properly checked out"
+    unless (-d srctop_dir()."/tlsfuzzer" && -d srctop_dir()."/tlsfuzzer/tests");
```

Comment:
I've seen the syntax srctop_dir("tlsfuzzer", "tests") elsewhere, I'd suggest to use that syntax

Reply:
Thanks! Done.